controllers/auction.js

```diff
@@ -49,6 +50,12 @@ const createNewAuction = async (req, res) => {
   try {
     const { id: seller } = req.userData
     const { initial_price: initialPrice, item_type: itemType, end_time: endTime, quantity } = req.body
+
+    const { currencies } = await wallet.fetchWallet(seller)
+    const itemQuantity = parseInt(currencies[`${itemType}`])
```

Comment:
Reason for ``currencies[`${itemType}`]``: https://stackoverflow.com/questions/51273469/how-to-resolve-eslint-generic-object-injection-sink-error